models/stg_shopify.yml

```diff
@@ -1542,7 +1542,7 @@ models:
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - order_id
-            - code
+            - index
```

Comment:
Is there a reason we removed `code` here? I can see from the ERD that `code` is not listed as a unique key. However, do we know why this was ever included?

Reply:
If we decide to move forward with removing `code` then we should include this in the CHANGELOG.

Reply:
I believe `code` was included because we were initially keeping the `index=1` filter. I'd be surprised if this was actually needed, as the `index=1` filter meant that only one code for one order would be present.

It isn't really clear from the release notes why `code` was used though, so I didn't feel like it was needed.  

Agreed on updating the CHANGELOG (which has been updated!).